src/Resources/PaymentTypes/ClickToPay.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace UnzerSDK\Resources\PaymentTypes;
+
+use UnzerSDK\Adapter\HttpAdapterInterface;
+use UnzerSDK\Resources\PaymentTypes\BasePaymentType;
+use UnzerSDK\Traits\CanAuthorize;
+use UnzerSDK\Traits\CanDirectCharge;
+
+class ClickToPay extends BasePaymentType
+{
+
+    use CanDirectCharge;
+    use CanAuthorize;
+
+    protected $mcCorrelationId;
+    protected $mcCxFlowId;
+    protected $mcMerchantTransactionId;
+    protected $brand;
+
+
+    public function __construct(
+        ?string $mcCorrelationId = null,
+        ?string $mcCxFlowId = null,
+        ?string $mcMerchantTransactionId = null,
+        ?string $brand = null
+    )
+    {
+        $this->mcCorrelationId = $mcCorrelationId;
+        $this->mcCxFlowId = $mcCxFlowId;
+        $this->mcMerchantTransactionId = $mcMerchantTransactionId;
+        $this->brand = $brand;
+    }
+
+    public static function getResourceName(): string
+    {
+        return 'clicktopay';
```

Comment:
I noticed one more thing.
This override doesn't actually ave the expected. resource in URI will still be determined based on the class name. Right now that would result in "click-to-pay"
That means this method is redundant and should is better to be removed. Class name should be "Clicktopay" so that it will result in "clicktopay" spelling.

Reply:
Updated